File: 프로그래머스/lv1/12912. 두 정수 사이의 합/두 정수 사이의 합.py
def solution(a, b):
    answer = 0
    while(a!=b):
        if(a>b):
            answer+=a
            a-=1
        elif(a<b):
            answer+=a
            a+=1
        else:
            answer=a
            return answer
    answer+=b
    return answer

File: 프로그래머스/lv1/12912. 두 정수 사이의 합/두 정수 사이의 합.java
class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        if(a==b) return answer=a;
        else{
            while(a!=b){
                if(a<b){
                    answer+=a;
                    a++;
                }
                else{
                    answer+=a;
                    a--;
                }
            }  
            answer+=b;
        }
        return answer;
    }
}